provider/consumer.py

```diff
@@ -526,3 +526,14 @@ def __on_revoke(self, consumer, partitions):
 
     def __errorOnJSONConstant(self, data):
     	raise(ValueError('Invalid JSON detected.'))
```

Comment:
how about providing a more detailed exception message similar to the ones from `__parseFloat`

Reply:
@mdeuser, done.